music_album.rb

```diff
@@ -3,7 +3,7 @@
 class MusicAlbum < Item
   DEFAULT_BOOL = false
 
-  attr_accessor :publish_date
+  attr_accessor :genre, :publish_date
 
   def initialize(genre, author, label, publish_date, on_spotify = DEFAULT_BOOL)
```

Comment:
Hi Ruben, You have done a great job. I just want to let you know that we have changed the parameters of initialize method in the parent class `Item`. I think it should be changed in The child class `MusicAlbum` too.

Reply:
Hi @medaminedev66

Already working on those changes. 👍